File: src/main/kotlin/net/starlegacy/feature/gas/collectionfactors/OutdoorsFactor.java
package net.starlegacy.feature.gas.collectionfactors;

import net.starlegacy.util.LegacyBlockUtils;
import org.bukkit.Location;

public class OutdoorsFactor extends CollectionFactor {
    @Override
    public boolean factor(Location location) {
        return !LegacyBlockUtils.INSTANCE.isInside(location, 2);
    }
}

File: server/src/main/kotlin/net/horizonsend/ion/server/features/gas/collectionfactors/OutdoorsFactor.kt
package net.horizonsend.ion.server.features.gas.collectionfactors

import net.horizonsend.ion.server.miscellaneous.utils.LegacyBlockUtils
import org.bukkit.Location

class OutdoorsFactor : CollectionFactor() {
    override fun factor(location: Location): Boolean {
        return !LegacyBlockUtils.isInside(location, 2)
    }

	override fun canBeFound(location: Location): Boolean = factor(location)
}
